PyFry.py

```diff
@@ -12,54 +13,58 @@
       -> Detecting eye coordinates and applying the deepfry eye flare in the center::DONE
 
 '''
-def userInput():
-    #Allowing user to choose the image that has to be deepfried
-    root = tk.Tk()
-    root.withdraw()
-    global filepath
-    filepath = list(root.tk.splitlist(filedialog.askopenfilenames(title="PyFry - Choose Image")))
-    print("picture location = ",filepath)
-
-    
+# def userInput():
+#     #Allowing user to choose the image that has to be deepfried
+#     root = tk.Tk()
+#     root.withdraw()
+#     global filepath
+#     filepath = list(root.tk.splitlist(filedialog.askopenfilenames(title="PyFry - Choose Image")))
+#     print("picture location = ",filepath)
+
+
 def irisCoords(eye):
-    #Finding the center point of the eye using the average outer extremes average of the eyes
-    mid = (eye[0] +eye[3])/2
+    # Finding the center point of the eye using the average outer extremes average of the eyes
+    mid = (eye[0] + eye[3])/2
```

Comment:
Don't make aesthetic changes to the working code. It creates more diffs and takes extra time to review.

Reply:
hi ionicc, Actually While adding my changes somewhere I got indentation error.
I tried to fix it but couldn't, then copy-pasted whole content in sublime-text and fixed indentation and put that back.

So,these kind of changes came in.  😃 😄 
I didn't check that much while commiting.
Will take care of these changes and raise a new PR.